docs/changelog.md

```diff
@@ -87,6 +87,10 @@ For more information on features and bug fixes in 1.0.0, see the GitHub mileston
 
 We are dropping support for HBase 1.
 
+##### Drop support for Solr 7
+
+We are dropping support for Solr 7.
+
```

Comment:
I probably missed the reason behind dropping Solr 7 support. I don't mind it but I think it would be great to describe the reason behind dropping support for Solr 7

Reply:
Solr 7 isn't really supported anymore and Solr 7 tests aren't stable. 

https://solr.apache.org/downloads.html

Reply:
Oh, I missed that. Thank you!